Sources/TSCBasic/Path.swift

```diff
@@ -707,6 +724,10 @@ private struct UNIXPath: Path {
 
     init(validatingRelativePath path: String) throws {
       #if os(Windows)
+        guard path != "" else {
+            self.init(normalizingRelativePath: path)
+            return
+        }
```

Comment:
"" is always a valid relative path. I don’t think there’s any changes. Adding the judgement here simply because retrieving `fileSystemRepresentation` of a blank string will halt the whole program.

Reply:
Ah, I see.  This deserves a comment I think.